src/scss/components/Workbench.scss

```diff
@@ -45,8 +45,7 @@
 }
 
 .workbenchWrapper {
-  width: inherit;
-  flex: 1 1 auto;
+  // TODO:
```

Comment:
What is the todo here? Can we remove this CSS class from this file if it is empty?

Reply:
Removed it.